dotnet/src/Planners/Planners.Core/Utils/FunctionUtils.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+internal static class FunctionUtils
+{
+    internal static void GetFunctionCallbackNames(string pluginFunctionName, out string pluginName, out string functionName)
```

Comment:
Did not give naming hee much thought.

Reply:
Suggestion: GetPluginAndFunctionNamesFromPlanData